src/chatd.cpp

```diff
@@ -2346,6 +2510,13 @@ void Chat::rejectMsgupd(Id id, uint8_t serverReason)
         throw std::runtime_error(errorMsg);
     }
 
+    // Update with contains meta has been reject by server. We don't notify
+    if (msg.type == Message::kMsgContainsMeta)
```

Comment:
Don't we need here to delete the msg from the sending queue? otherwise, the edit will be retried sooner or later.